build.assets/macos/tctl/tctl.app/Contents/PkgInfo

```diff
@@ -0,0 +1 @@
+APPL????
```

Comment:
Is this correct contents of this file?

Reply:
Yeah this is correct. It's some legacy format with `APPL` which defines it as an application bundle and `????` used to be a unique signature which is no longer supported.

I'm not actually sure if it's a required file though but our `tsh.app` already had it so I figured I'd just keep it. It might also get autogenerated anyway since all apps I looked at under `/Applications` have that file that looks like that as well.